docs-new/.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: ci 
+on:
+  push:
+    branches:
+      - master 
+      - main
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          git fetch --tags
+          git worktree add home/docs/1.3.1 docs-1.3.1
+          git worktree add home/javadoc javadoc
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - run: echo "cache_id=$(date --utc '+%V')" >> $GITHUB_ENV 
+      - uses: actions/cache@v3
+        with:
+          key: mkdocs-material-${{ env.cache_id }}
+          path: .cache
+          restore-keys: |
+            mkdocs-material-
```

Comment:
I think some cache id is missing here. I would suggest leaving out the caching initially.

Reply:
lol oops :) yes